src/systems/filecoin_mining/sector/sector.go

```diff
@@ -1,20 +1,15 @@
 package sector
 
-import actor "github.com/filecoin-project/specs/systems/filecoin_vm/actor"
 import block "github.com/filecoin-project/specs/systems/filecoin_blockchain/struct/block"
 
 // TODO: placeholder epoch value -- this will be set later
 // ProveCommitSector needs to be submitted within MAX_PROVE_COMMIT_SECTOR_EPOCH after PreCommit
 const MAX_PROVE_COMMIT_SECTOR_EPOCH = block.ChainEpoch(3)
 
-func (r *FaultReport_I) GetDeclaredFaultSlash() actor.TokenAmount {
-	return actor.TokenAmount(0)
-}
+type StorageFaultType int
 
-func (r *FaultReport_I) GetDetectedFaultSlash() actor.TokenAmount {
-	return actor.TokenAmount(0)
-}
-
-func (r *FaultReport_I) GetTerminatedFaultSlash() actor.TokenAmount {
-	return actor.TokenAmount(0)
-}
+const (
+	DeclaredFault   StorageFaultType = 0
```

Comment:
minor nit: prefer to start enums at 1, to avoid accidental uninitialized errors

```
const (
    CaseA EnumType = 1+iota
    CaseB
    CaseC
    // ...
)
```